src/terminal/parser/ft_fuzzwrapper/main.cpp

```diff
@@ -70,7 +70,10 @@ int __cdecl wmain(int argc, wchar_t* argv[])
             fGotChar = GetChar(&wch);
         }
 
-        fclose(hFile);
+        if (hFile != NULL)

```

Comment:
per ES.87: Don’t add redundant == or != to conditions ISO C++ guideline:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-if

It isn't necessary to check against NULL for pointer; you can simplify this to `if (hFile)`.

Reply:
Thanks for your suggestion. Improved.